source/developers/projects/studio/api/workflow/go-delete.rst

```diff
@@ -28,15 +28,17 @@ Resource Information
 Parameters
 ----------
 
-+---------------+-------------+---------------+--------------------------------------------------+
-|| Name         || Type       || Required     || Description                                     |
-+===============+=============+===============+==================================================+
-|| site_id      || String     || |checkmark|  || Site to use                                     |
-+---------------+-------------+---------------+--------------------------------------------------+
-|| user         || String     || |checkmark|  || User executing operation                        |
-+---------------+-------------+---------------+--------------------------------------------------+
-|| items        || String     || |checkmark|  || Array of paths to delete                        |
-+---------------+-------------+---------------+--------------------------------------------------+
++--------------------+-------------+---------------+--------------------------------------------------+
+|| Name              || Type       || Required     || Description                                     |
++====================+=============+===============+==================================================+
+|| site_id           || String     || |checkmark|  || Site to use                                     |
++--------------------+-------------+---------------+--------------------------------------------------+
+|| user              || String     || |checkmark|  || User executing operation                        |
++--------------------+-------------+---------------+--------------------------------------------------+
+|| items             || String     || |checkmark|  || Array of paths to delete                        |
++--------------------+-------------+---------------+--------------------------------------------------+
+|| submissionComment || String     ||              || submission comment                              |
```

Comment:
You named this `comment` in Bulk Go Live. Let's keep it consistent and name this `comment` like that one: https://docs.craftercms.org/en/3.1/developers/projects/studio/api/deployment/bulk-golive.html